<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dissolve</class>
 <widget class="QDialog" name="Dissolve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dissolve</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>3</y>
     <width>600</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Input</string>
   </property>
   <widget class="QPushButton" name="pushButtonInput">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>98</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Browse the polygon file you want to dissove!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPolygonInput">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>34</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>34</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>A MULTI-polygon shapefile (multiple watershed boundary)
that needs to be dissolved to one polygon shapefile.</string>
    </property>
    <property name="text">
     <string>Input Polygon File</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>78</y>
     <width>600</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Output</string>
   </property>
   <widget class="QPushButton" name="pushButtonOutput">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>98</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Browse a file name to which dissolve file will be saved!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>37</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Dissolved polygon shapefile</string>
    </property>
    <property name="text">
     <string>Output Dissolve File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPolygonOutput">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>34</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>168</y>
     <width>600</width>
     <height>47</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButtonHelp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>98</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClose">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>9</y>
      <width>98</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRun">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>9</y>
      <width>98</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Run!</string>
    </property>
   </widget>
  </widget>
  <widget class="Q3TextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>227</y>
     <width>600</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
